OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -122,62 +132,117 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">8</dim>
                                         <dim type="left">4</dim>
                                     </dimensions>
+                                    <note>The characters per line are 10–12.</note>
+                                    <ab type="pricking">Ruling and pricking are barley visible.</ab>
+                                    <ab type="ruling" subtype="pattern">
+                                        <locus from="2r" to="15v"/>Ruling pattern:
+                                        1A-1A/0-0/0-0/C.</ab>
+                                    <ab type="pricking">Primary pricks are visible.</ab>
+                                    <ab type="pricking">Ruling pricks are not visible.</ab>
                                 </layout>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <locus from="2r" to="15v"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1800" notAfter="1950">19th/20th century</date>
```

Comment:
Do we use 19th-20th or 19th/20th? I've always used the hyphen so far. I do not care between the two, it's just a matter of coherence.

Reply:
For me the hyphen refers to a range. I wanted to express the uncertainty: whether 19th or 20th.
Maybe its better to wroite like this? `19th or 20th century?`

Reply:
19th/20th is fine for me. Let's just see if there are objections from @thea-m @Gnisci or @ElianaDalSasso . If we adopt that way, I'll correct my previous records

Reply:
I have no opinion, 19th/20th seems good to me

Reply:
For me 19th-20th expresses a range 19th/20th an alternative. Could also write `19th to 20th centuries.` to express the first and `19th or 20th century.` to express the second (note the difference between plural for the first and singular for the second.